src/shared/mount-util.c

```diff
@@ -875,7 +875,7 @@ static int mount_in_namespace_legacy(
         mount_tmp_created = true;
 
         if (is_image)
-                r = verity_dissect_and_mount(chased_src_fd, chased_src_path, mount_tmp, options, image_policy, NULL, NULL, NULL, NULL);
+                r = verity_dissect_and_mount(chased_src_fd, chased_src_path, mount_tmp, options, image_policy, NULL, NULL, NULL, NULL, NULL);
```

Comment:
you touched it you fix it ;-)

i.e please add `/* ret_image= */` comments to the NULL args (and break the line)